contrib/automation_tests/test_cases/live_tab.py

```diff
@@ -19,13 +19,14 @@ def __init__(self, **kwargs):
         self._live_tab = None
 
     def execute(self, suite):
+        self.find_control("TabItem", "Live", parent=suite.top_window()).click_input()
         self._live_tab = self.find_control("Group", "liveTab", parent=suite.top_window())
         super().execute(suite)
 
-    def find_function_cell(self, function_name) -> Tuple[BaseWrapper, int]:
+    def find_scope_cell(self, scope_name) -> Tuple[BaseWrapper, int]:
```

Comment:
The overload of the term "scope" is a bit unfortunate here. It took several minutes for me to understand that a scope can be a function call or a manual instrumentation scope. A comment here and / or somewhere in `VerifyScopeTypeAndHitCount` might help.

Also: I'm not exactly a python wizard but I think this method should be named `_find_scope_cell` to encourage protected use only.

Reply:
The term "scope" isn't introduced here. It's now heavily used in the codebase. But I'll add a comment.